From 5fa09685561e8673357d8aa71c889a76b3f9546b Mon Sep 17 00:00:00 2001
From: Josh Wu <josh.wu@atmel.com>
Date: Tue, 10 Feb 2015 14:42:34 +0800
Subject: [PATCH] Makefile: fs/src/fat.mk: use simpler format to include file
 to build

using this format to include obj file, which is simple:
	OBJS-$(CONFIG_xxx) += xxx.o

Signed-off-by: Josh Wu <josh.wu@atmel.com>
---
 fs/src/fat.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/fs/src/fat.mk b/fs/src/fat.mk
index b835b4c..5cce2b5 100644
--- a/fs/src/fat.mk
+++ b/fs/src/fat.mk
@@ -5,8 +5,6 @@ FS_FAT:=$(TOPDIR)/fs/src
 
 COBJS-$(CONFIG_FATFS)	+=  $(FS_FAT)/ff.o
 COBJS-$(CONFIG_FATFS)	+=  $(FS_FAT)/diskio.o
-ifeq ($(CONFIG_LONG_FILENAME), y)
-COBJS-$(CONFIG_FATFS)	+=  $(FS_FAT)/option/ccsbcs.o
-endif
+COBJS-$(CONFIG_LONG_FILENAME)	+=  $(FS_FAT)/option/ccsbcs.o
 
 
-- 
1.9.1

